### DISCUSSION TITLE
error when I save a new events

### DISCUSSION THREAD

**Discussion by dlaflamme1** [liedekef/events-made-easy#255] [Q&A]
I have this error with my database when I try to create new event
. Someone know what can cause that?
thanks in advance

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=96

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=97

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=98

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=99

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=100

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=101

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=102

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=103

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=104

Erreur de la base de données WordPress : [Table 'piucc062_wp700.wpkh_eme_task_signups' doesn't exist]
DELETE FROM wpkh_eme_task_signups WHERE event_id=105

**Comment by liedekef** [comment]
That means the table eme_task_signups (with prefix wpkh_) doesn't exist. That table is being created if EME has database changes, so if it is not existing on your end, something blocked your wordpress database config/user from creating this table.
For forcing the table to be recreated (if possible), go in the EME Settings menu, tab "Other" and save the value for the option "EME DB version" from 371 (which should be the current version) to 370 and press save. Upon saving, EME will try to create the table again (and update the value from "EME DB version" to 371 again, so don't be surprised if the value seems to be the same again).
If the table is not created, your wordpress database user has issues creating the table and should be checked.
